CHANGELOG.md

```diff
@@ -30,6 +30,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Changed
 
 - Do not create Dockerfile.* in application's directory.
+- Updated `tarantool/luarocks` dependency to `v3.10.0`.
```

Comment:
It will be more clear for users I think.
```suggestion
- Updated `luarocks` to `v3.10.0`.
```

Reply:
changed